.github/workflows/haskell-ci.yml

```diff
@@ -17,6 +17,14 @@ on:
         type: string
         required: false
         default: latest
+      cache-key-prefix:
+        description: |
+          Prefix to add to cache key(s).
+
+          May be used to distinguish caches for multiple packages in the same repo.
+        type: string
+        required: false
+        default: ""
```

Comment:
This is also what you'd change if you want to invalidate the cache à la `CACHE_VERSION`, right? That might be worth documenting as well.